src/SDKs/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\git\azure-sdk-for-net\src\SDKs
```

Comment:
There should be only 1 .txt file modified. Please checkout both .txt files and regenerate the code when the spec is merged.